src/main/java/org/mockito/testng/MockitoTestNGListener.java

```diff
@@ -63,6 +63,8 @@
     private final MockitoBeforeTestNGMethod beforeTest = new MockitoBeforeTestNGMethod(sessions);
     private final MockitoAfterTestNGMethod afterTest = new MockitoAfterTestNGMethod(sessions);
 
+    public static final String STRICTNESS_LEVEL = "org.mockito.testng.strictness";
```

Comment:
Javadoc. In my opinion the java doc could point to a section in the Javadoc of `MockitoTestNGListener` (currently there's no section mentioning the setting, that would be better here than in the README, because it should be accessible right in the IDE).

Reply:
I wonder if this constant needs to be public, e.g. what if later this constant needs to be moved in another class, like `TestNGParamters`.

Reply:
I exposed the constant because I started with XML usage and then shoehorned annotation usage, but I can make it private if take your helper method suggestion.  This would cause a regression in the XML case which can remedied (didn't think of this initially) with an XML Schema or DTD.  I'm leaning towards XML Schema because it can be generated from JAXB annotations whereas DTD afaik cannot.  Thoughts?